atsign-foundation/noports
# fix: key management dialogs updated to allow text to scale based on d…

…evice size and fixed size given to the dialogs.



**- What I did**

- Key Management Dialogs given fixed size
- Key Management Dialogs text size changes as device size changes.


**- How I did it**

**- How to verify it**

1. Navigate to Settings
2. Click the SSH Key Management List tile
3. Change the device size and observe the changes to the text size.

**- Description for the changelog**

Key Management Dialogs updated to improve adaptability.
